### DISCUSSION TITLE
Feature Request: Secrets GUI, allow masking real secrets and show normal variables

### DISCUSSION THREAD

**Discussion by joaduo** [community/community#17277] [Actions]
Hi! 

Consider the following 2 example Variables (Secrets)
```
AWS_ACCESS_KEY_ID = AKIARAY....213
AWS_SECRET_ACCESS_KEY = a8923898u....as09d0a9s
```

I need to be able to see the contents of `AWS_ACCESS_KEY_ID` but not the contents of `AWS_SECRET_ACCESS_KEY` because the later is the real secret.

Other platforms allow setting when a variable is a secret (masking/hiding them) and when it can be shown in the Web GUI. (for auditing or debugging purposes)
In the example above of AWS you want to know which KEY ID is being used (so you can know which privileges that repo has)

Note that this feature maybe would cover this other request https://github.com/github/feedback/discussions/9969

**Comment by jsoref** [comment]
Fwiw, if you have something that isn't truly a secret, you could store it w/ `rot13` encoding and then decode it at runtime.

```sh
echo AWS_ACCESS_KEY_ID = $(echo AKIARAY....213|tr 'A-Za-z' 'N-ZA-Mn-za-m')
AWS_ACCESS_KEY_ID = NXVNENL....213
```
